### PR TITLE
Change site to use typewind

### PR DESCRIPTION
**Pull request by guilhermefront**
Attempt to use the library on the own website. Making it work will be a great display, but will require solving some crucial bugs happening. Currently it breaks with the following:

image

### CODE REVIEW

**Comment by NicolasLopes7** [comment]
this gave me the idea to write a codemod, from tailwind to typewind 🤔. I'll write an issue soon

**Comment by Mokshit06** [comment]
hey! thanks I'll look into these issues

File: site/components/Home.tsx [hunk 1]
```diff
@@ -1,29 +1,65 @@
...
+    <main
+      className={
+        tw.my_6.max_w_['75rem'].flex.flex_col.px_4.mx_auto.items_center
+      }
+    >
+      <div className={tw.w_full.max_w_['53rem']}>
+        <h1
+          className={
+            tw.block.text_center.font_extrabold.text_['4xl']
+              .sm(tw.text['5xl'])
```
Comment (Mokshit06) [inline site/components/Home.tsx:16]: This line was the one causing the error. It should be `tw.text_['5xl']` instead. I have made the errors more descriptive now, so it should also report where the error occurred

**Comment by Mokshit06** [comment]
Some styles were different from the original website so I've fixed those and formatted them correctly. Merging 👍

**Pull request merged**